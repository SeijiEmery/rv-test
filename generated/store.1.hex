0000000    23  30  a3  00  23  24  a3  00  23  18  a3  00  23  0c  a3  00
0000020    83  35  03  00  03  36  83  00  83  36  03  01  03  37  83  01
0000040    13  00  00  00  13  00  00  00  13  00  00  00  13  00  00  00
0000060    13  00  00  00  73  00  10  00                                
0000070
