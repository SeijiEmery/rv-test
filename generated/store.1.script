load /x 0 generated/store.1.hex
writereg 10 255
setpc 0
run 100000
readreg 10
readreg 11
readreg 12
readreg 13
readreg 14