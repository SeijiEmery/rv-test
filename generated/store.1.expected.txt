tests/store.test.s:22
R10 = 255
R11 = 255
R12 = 255
R13 = 255
R14 = 255
